lib/ansible/modules/web_infrastructure/pmp.py

```diff
@@ -0,0 +1,296 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# (c) 2017, Bernat Mut <bernat.mut@aireuropa.com>
+#
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+
+__metaclass__ = type
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+DOCUMENTATION = """
+module: pmp
+version_added: "2.5"
+short_description: Create and modify accounts and groups in Password Manager Pro (PMP)
+description:
+  - Create and modify accounts and groups in Password Manager Pro (PMP).
```

Comment:
Maybe a little more information about what password manager pro would be worthwhile to have here.